django/api/templates/admin/api/initiatedgoelectricrebateapplication/submit_line.html

```diff
@@ -0,0 +1,23 @@
+{% extends "admin/submit_line.html" %}
+{% load i18n admin_urls jazzmin %}
+{% get_jazzmin_ui_tweaks as jazzmin_ui %}
+{% block submit-row %}
+<div class="card card-primary card-outline">
+  <div class="card-header">
+    <h3 class="card-title">
+      <i class="fas fa-edit"></i>
+      {% trans 'Actions' %}
+    </h3>
+  </div>
+  <div class="card-body">
+    <div class="form-group">
+      <input
+        type="submit"
+        class="btn {{ jazzmin_ui.button_classes.success }} form-control"
```

Comment:
In order to be consistent with what you're doing in the "message_user()" function in admin.py, the class should be ""btn {{ jazzmin_ui.button_classes.danger }} form-control"